packages/create-plugin/src/utils/utils.templates.ts

```diff
@@ -96,10 +96,10 @@ export function renderTemplateFromFile(templateFile: string, data?: any) {
 export function getTemplateData(data?: Partial<TemplateData>): TemplateData {
   const { features } = getConfig();
   const currentVersion = getVersion();
-  const pluginJson = getPluginJson();
-  const pluginType = data?.pluginType ?? pluginJson.type;
-  const packageManagerInfo = getPackageManagerWithFallback();
-  const packageManagerName = data?.packageManagerName ?? packageManagerInfo.packageManagerName;
+  const pluginJson = data ? undefined : getPluginJson();
```

Comment:
Hmm, I am not sure that we should do this to be honest - like this we remove the possibility to only specify (override) certain template variables using the `data` param (now in case the call-site only wants to specify the `pluginName`, the other properties will fall back to `pluginJson?.`, which will always be undefined).

Reply:
When it is just `const pluginJson = getPluginJson();` and we create a new plugin it is failing because it is executing getPluginJson function anyway and can not find it. And I see the error "Make sure you run this command from a plugins root directory."

Reply:
In that case we should try fetching the plugin json here in a resilient way (with a fallback). Although these changes indirectly fix this problem of the not existing plugin json, they now make it necessary for the call-site to specify all the attributes in the data (as if they override a single one, the pluginJson will be set to undefined and they cannot rely on it as a default anymore).

Reply:
After thinking about this for a few more times, I now actually think that I was wrong suggesting this function to be more generic, and accept a partial of the `TemplateData` types. (Making it generic in a sense that it can accept an override of even a single property made it quite complicated, mostly due to the fact that the plugin.json and package.json files are not always available as a fallback.)

Summing it up please ignore my comments above, and I will resolve this thread now.